### ISSUE TITLE
document "Source Variables" in Source page

### ISSUE THREAD

**Issue by jwlogemann** [ansible/awx#6538]
##### ISSUE TYPE
 - Bug Report

##### SUMMARY
The "source variable" field is basically undocumented now. There is a popup that links to an ec2.ini file, which returns a 404. 

##### ENVIRONMENT
* AWX version: 9.1.0
* AWX install method: docker on linux
* Ansible version:  2.8
* Operating System: Centos 7.7
* Web Browser: Firefox

##### STEPS TO REPRODUCE

Go to source page, click on question mark next to source variables

##### EXPECTED RESULTS

Some useful information on how to use this field

##### ACTUAL RESULTS

a popup linking to a 404

**Comment by kdelee** [comment]
@AlanCoding I see the patch that @ryanpetrello landed to update to "current state" but I think I'm going to leave this open until your `vendoring_collections` PR lands and we can point to the docs appropriate for that state of the world.

https://github.com/ansible/awx/pull/6283 for reference

**Comment by kdelee** [comment]
- [x] ec2, updated to link to https://github.com/ansible-collections/community.aws/blob/master/scripts/inventory/ec2.ini
![ec2_source_vars](https://user-images.githubusercontent.com/29239279/80397480-2c637e80-8884-11ea-963e-c3ddd4913bee.png)

- [x] azure updated to link to https://github.com/ansible-collections/community.general/blob/master/scripts/inventory/azure_rm.ini
![azure_source_vars](https://user-images.githubusercontent.com/29239279/80397488-308f9c00-8884-11ea-8514-053b8707dcf9.png)

- [x] vmware updated to link to https://github.com/ansible-collections/vmware/blob/master/scripts/inventory/vmware_inventory.ini
![vmware_source_vars](https://user-images.githubusercontent.com/29239279/80397494-34bbb980-8884-11ea-8640-1129c71c5a03.png)

- [ ] NOT DONE: openstack also has source vars, incorrectly links to 404 page https://github.com/ansible/ansible/blob/devel/contrib/inventory/openstack.yml
![openstack_source_vars](https://user-images.githubusercontent.com/29239279/80397507-38e7d700-8884-11ea-8543-5028604a1105.png)

- [ ] NOT DONE: foreman/sat6 also has source vars, incorrectly links to 404 page https://github.com/ansible/ansible/blob/devel/contrib/inventory/foreman.ini
![foreman_source_vars](https://user-images.githubusercontent.com/29239279/80397514-3c7b5e00-8884-11ea-81fb-cd6df2cba06e.png)

- [ ] NOT DONE: cloudforms also has source vars, incorrectly links to 404 page: https://github.com/ansible/ansible/blob/devel/contrib/inventory/cloudforms.ini
![coudforms_source_vars](https://user-images.githubusercontent.com/29239279/80397520-400ee500-8884-11ea-8f3d-0b10ec5cd264.png)


@AlanCoding we need one more PR to fix this

**Comment by kdelee** [comment]
@rebeccahhh @ryanpetrello @AlanCoding taking another look at this I'm actually confused as to why we are linking to the source vars for the scripts if we are now using the plugings

Because, yes, the scripts live in the same collection that the plugins do

:sos:

**Comment by AlanCoding** [comment]
That is https://github.com/ansible/awx/issues/4306, replaced by https://github.com/ansible/awx/issues/5150

> I'm actually confused as to why we are linking to the source vars for the scripts if we are now using the plugings

the source variables in these cases take options as they were formatted for the scripts. The server logic looks for those same options, otherwise backward compatibility would be broken.

We should also pull in `ansible-doc` data for each plugin and feed that into a form builder in the UI... but if we ignore that it's just a different way to interpret the YAML, and no option is made available for the user now.

**Comment by kdelee** [comment]
:grimacing: Ok. well. I guess I accept the way this is...

except this has been merged downstream and I'm not finding this code in awx...so I think we need to port it to awx before we can close this out. looks fine downstream

**Comment by kdelee** [comment]
@ryanpetrello what say you

**Comment by kdelee** [comment]
OK, it was settled, we DO want to point to the old script files because we are still supporting that interface, we will be updating to New World Order config interface at a later time. Also code is now been merged into devel, so closing
